test/setup

```diff
@@ -0,0 +1,96 @@
+set -e
+
+
+exec {BASH_XTRACEFD}>&1
+echo "$0" "$@"
+set -x
+
+
+test -n "$MPV_MPRIS_TEST_NAME" ||
+exec "./env" "$0" "$@"
+
+
+test="$(basename "$0")"
+
+
+if [ "$test" = setup ] ; then
+	echo "This setup script should be sourced, not executed" >&2
+	exit 1
+fi
+
+
+input_json="$MPV_MPRIS_TEST_MPV_IPC/$test.mpv.ipc.input.json"
+output_json="$MPV_MPRIS_TEST_MPV_IPC/$test.mpv.ipc.output.json"
+ipc="$MPV_MPRIS_TEST_MPV_IPC/$test.mpv.ipc"
+log_prefix="$MPV_MPRIS_TEST_LOG/$test"
+mpv_log="$log_prefix.mpv.log"
+socat_log="$log_prefix.socat.log"
+file="$MPV_MPRIS_TEST_PLAY"
+
+
+prop () {
+	prop="$1"
+	shift
+	jo command[]=get_property command[]="$prop" "$@"
```

Comment:
Can we use jq instead of jo to remove one of the test dependencies here?

```suggestion
	jq -n -c --arg prop "$1" '{command: ["get_property", $prop]}'

```

Reply:
Fixed.